one/src/query.rs

```diff
@@ -0,0 +1,56 @@
+use clap::Args;
+use datafusion_cli::{exec::exec_from_repl, print_options::PrintOptions};
+use object_store::aws::AmazonS3Builder;
+
+#[derive(Args, Debug)]
+pub struct QueryOpts {
+    /// Endpoint of a Flight SQL server for the conclusion feed.
+    #[arg(
+        short,
+        long,
+        default_value = "http://127.0.0.1:5102",
+        env = "CERAMIC_ONE_FLIGHT_SQL_ENDPOINT"
+    )]
+    flight_sql_endpoint: String,
+
+    /// AWS S3 bucket name.
+    /// When configured the aggregator will support storing data in S3 compatible object stores.
+    ///
+    /// Credentials are read from the environment:
+    ///
+    ///   * AWS_ACCESS_KEY_ID -> access_key_id
+    ///   * AWS_SECRET_ACCESS_KEY -> secret_access_key
+    ///   * AWS_DEFAULT_REGION -> region
+    ///   * AWS_ENDPOINT -> endpoint
+    ///   * AWS_SESSION_TOKEN -> token
+    ///   * AWS_ALLOW_HTTP -> set to "true" to permit HTTP connections without TLS
+    ///
+    #[arg(long, env = "CERAMIC_ONE_AWS_BUCKET")]
+    aws_bucket: String,
```

Comment:
kind of the inverse of my above comment.  AWS is a specific platform, one we are notably not using.  Should this say be s3_bucket or something like that?  We're using the S3 API, but we aren't using the AWS platform...